code/game/objects/items/stacks/packagewrap.dm

```diff
@@ -195,7 +197,11 @@
 
 /obj/item/delivery/large/attack_hand(mob/user as mob)
 	if(wrapped)
-		wrapped.forceMove(get_turf(src.loc))
+		wrapped.forceMove(get_turf(loc))
```

Comment:
#10877
TL;DR `get_turf()` now uses an internal BYOND function to get the turf and it's absurdly fast